Shadowlands/TazaveshTheVeiledMarket/Soleah.lua

```diff
@@ -113,3 +118,8 @@ function mod:EnergyFragmentation(args)
 	self:Message(args.spellId, "orange")
 	self:PlaySound(args.spellId, "alert")
 end
+
+function mod:ShurikenBlitz(args)
+	self:Message(args.spellId, "red", CL.casting:format(args.spellName))
+	self:PlaySound(args.spellId, "warning")
```

Comment:
The warning sound should really be for mission critical stuff. Here it seems like this could be spammy, and afaik only interrupters care about this ability?

Reply:
It can also be stunned, knocked back, disoriented, feared, etc. So almost every role and class has something to deal with it. And it's pretty much a wipe if this cast goes off on Tyrannical in any higher key level.